src/styles/Message.scss

```diff
@@ -986,3 +986,61 @@
     max-width: 240px;
   }
 }
+
+.str-chat {
+  // fixes the overall overflow/flex issues together with the rule below
+  .mml-container {
+    display: flex;
+    padding-bottom: 4px;
+  }
+
+  // fixes the overall overflow/flex issues together with the rule above
+  .mml-wrap {
+    display: block;
+    max-width: none;
+    // the max-width should match that to .str-chat__message-XXXX-text-inner
+    max-width: 345px;
+
+    @media screen and (max-width: 375px) {
+      max-width: 220px;
+    }
+  }
+
+  .mml-datepicker {
+    min-width: 180px;
+  }
+
+  // undo the negative margins that make the mml content to expand to the
+  // viewport edges
+  .mml-align-right .mml-btnlist--floating,
+  .mml-align-right .mml-carousel {
+    margin-left: auto;
+  }
+  .mml-align-left .mml-btnlist--floating,
+  .mml-align-left .mml-carousel {
+    margin-right: auto;
+  }
+
+  @media screen and (max-width: 960px) {
+    .mml-align-left .mml-carousel {
+      margin-left: -10px;
+    }
+    .mml-align-right .mml-carousel {
+      margin-right: -10px;
+    }
+  }
+
+  // expand carousel to the edge of the screen, the horizontal padding is theme
+  // specific in stream-chat-react
+  &.messaging {
+    // as .messaging.str-chat .str-chat__list:
+    @media screen and (max-width: 960px) {
+      .mml-align-left .mml-carousel {
+        margin-right: -10px;
```

Comment:
sorry this was a mistake of mine I fixed in a later commit, should be `margin-left` here @mahboubii

Reply:
thanks for the heads up